src/instance/discord/ChatManager.ts

```diff
@@ -64,15 +64,28 @@ export default class ChatManager extends EventHandler<DiscordInstance> {
     }
   }
 
-  async hasBeenMuted(event: Message, discordName: string, readableName: string): Promise<boolean> {
+  async truncateText(message: Message, content: string): Promise<string> {
+    const length = 250
+    if (content.length <= length) {
+      return content
+    }
+
+    await message.reply({
+      content: `Message too long! It has been shortened to ${length} characters.`
+    })
+
+    return content.slice(0, Math.max(0, length)) + '...'
```

Comment:
What is the point of `Math.max(0, length)`. It will always return `length`. 
```suggestion
    return content.slice(0, length) + '...'
```

Reply:
your linter did it, not me